bigquery/indexer.py

```diff
@@ -462,15 +427,21 @@ def main():
     bq_client = bigquery.Client(project=deploy_project_id)
     storage_client = storage.Client(project=deploy_project_id)
 
+    all_table_documents = {}
```

Comment:
Naming nits, I think the "all" isn't needed. Also changing some names to reflect what they're doing in this PR.

all_table_documents -> participant_docs
all_fields_documents -> field_docs

index_fields -> add_table_to_fields_docs
index_table -> add_table_to_participant_docs

_get_field_docs_and_add_to_dict -> _update_fields_docs_inner or _update_fields_docs
_get_sample_docs_and_add_to_dict -> _add_sample_table_to_participant_docs
_get_participant_docs_and_add_to_dict -> _add_participant_table_to_participant_docs

Instead of
```
        add_table_to_fields_docs(es, fields_index_name, table, sample_id_column,
                     field_docs)
```
I think this is a little cleaner:
```
        field_docs = add_table_to_fields_docs(es, fields_index_name, table, sample_id_column,
                     field_docs)
```
Same for similar methods.

Reply:
I like the name changes and I made them, but I don't understand what you're suggesting with 

>         field_docs = add_table_to_fields_docs(es, fields_index_name, table, sample_id_column,
>                      field_docs)

add_table_to_fields_docs doesn't return anything.

Reply:
I think it's a little cleaner if you change it to return field_docs. Then when we read the code, it's easier to tell that field_docs is being updated.